pkg/apis/gcp/v1alpha1/project_types.go

```diff
@@ -0,0 +1,91 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
Shouldn't we remove this comment?

Reply:
We have left this in on other repos, but I don't see why we shouldn't remove it.  Do you think we should just remove it ?

Reply:
I understand it as note by framework for us to remove once we understood this file is for us to touch ;)

Reply:
I will remove on all files in this operator